src/container/srv_target.c

```diff
@@ -74,6 +75,10 @@ cont_aggregate_epr(struct ds_cont_child *cont, daos_epoch_range_t *epr)
 	if (dss_ult_exiting(cont->sc_agg_req))
 		return 1;
 
+	rc = ds_obj_ec_aggregate(cont, epr);
+	if (rc)
+		D_ERROR("EC aggregation returned: "DF_RC"\n", DP_RC(rc));
+
```

Comment:
Sure, this is just a place holder call.

Reply:
Parallel execution of VOS aggregation and EC aggregation is problematic.  Once the coordination between the two processes (epoch ranges) is in place, this will be addressed.